### ISSUE TITLE
Unable to snapshot filter using @home

### ISSUE THREAD

**Issue by enly1** [digint/btrbk#318]
When attempting to use @home as a filter on a snapshot command, the subvolume name is not recognised.

**Command**: `sudo btrbk snapshot '@home'`

**Output**:
```
ERROR: Bad argument: not a subvolume/group declaration: @home
usage: btrbk [options] <command> [filter...]

options:
   -h, --help                display this help message
       --version             display version information
   -c, --config=FILE         specify configuration file
...
```
@home is the name of a subvolume in the config, so issues with the @ symbol I guess.

**Config Fragment:**
```
volume /mnt/timeshift/backup
  # no action if external disk is not attached
  #snapshot_create  ondemand

  #318 propagates to all subvolume sections:
  #target /media/dev/btrfs-ssd/laptop-snapshots

  group laptop

  subvolume @
    # use different retention policy for kvm backups
    target_preserve     7d 4w
    target send-receive /media/dev/btrfs-ssd/laptop-snapshots

  subvolume @home
    target_preserve     7d 4w
    target send-receive /media/dev/btrfs-ssd/laptop-snapshots
```

**Comment by digint** [comment]
I can't reproduce this on btrbk-0.29.0, this seems to be already fixed by commit f0cff5ee5acb3b853c6ea9f1440d7f1c99dc0b0e (included in v0.28.0). What version are you running?

Can you confirm that it is working on v0.29.0? Try:

```
cd /tmp/
wget https://raw.githubusercontent.com/digint/btrbk/v0.29.0/btrbk
chmod +x btrbk
./btrbk -n snapshot '@home'
```

If not, please provide a trace log (`btrbk -n snapshot -l trace 2>/tmp/btrbk_trace.log`)

**Comment by enly1** [comment]
I had used 0.29 from a download directory, but instead of moving it into system folders, I simply did a distro install assuming it was also the latest.  Turns out it was 0.26 that it installed.

Problem existed in 0.26, but works fine in 0.29.

Sorry - thought was already using latest, otherwise would have updated before reporting issue.
